artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/perf/PerfClientCommand.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.artemis.cli.commands.messages.perf;
+
+import javax.jms.ConnectionFactory;
+import javax.jms.Destination;
+import java.util.Queue;
+import java.util.concurrent.Executor;
+import java.util.concurrent.LinkedTransferQueue;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.LockSupport;
+
+import io.airlift.airline.Command;
+import io.airlift.airline.Option;
+import io.netty.channel.DefaultEventLoop;
+import io.netty.channel.DefaultEventLoopGroup;
+import io.netty.channel.EventLoop;
+import org.apache.activemq.artemis.cli.commands.ActionContext;
+
+@Command(name = "client", description = "It will produce and consume messages to a broker instance")
```

Comment:
I wonder if `client` is the best name here. It seems a little too close to the `consumer` and `producer` commands we already have. I realize that it's part of the `perf` suite of commands, but maybe something like `benchmark` or `run` might be better. Naming things is hard. :(

Reply:
Agree; I didn't called it `benchmark` because `perf benchmark` sounds like a repetition to me, but probably is better then `perf client`.

Consider that the `perf` tools are:
- client
- producer
- consumer

And indeed have duplicates of our existing `producer`/`consumer` tools although I've hoped that the `perf` prefix helped to distinguish them...let's speak online to find some better naming